src/Security/Security.php

```diff
@@ -1284,7 +1243,14 @@ public static function clear_database_is_ready()
      */
     public static function force_database_is_ready($isReady)
     {
-        self::$force_database_is_ready = $isReady;
+        $toForce = [
+            Member::class,
+            Group::class,
+            Permission::class,
+        ];
+        foreach ($toForce as $class) {
+            DB::force_database_is_ready($class, $isReady);
```

Comment:
as with `clear_database_is_ready()` - I think this is a subtle but significant behaviour change

Reply:
My thinking is the same as with `clear_database_is_ready` - but I'll make a change if you want me to. What change would you recommend here? I don't think it would be appropriate to force DB to consider _all_ classes as "ready".

Reply:
Ditto include Member subclasses

Reply:
Because this is only used for tests, and those tests don't need us to check subclasses of `Member` are ready, and `databaseIsReady` returns true if the class passed in is in `$databaseReadyClasses` (with the assumption that it has therefore already gone through the full hierarchy check), I don't think we need to add `Member` subclasses here.

This is also consistent with the old check:
```php
if (self::$force_database_is_ready !== null) {
    return self::$force_database_is_ready;
}
```